src/Server/RestRouterHandlers/DatabaseRestRouterHandler.cpp

```diff
@@ -13,12 +10,27 @@ namespace DB
 
 namespace
 {
-std::map<String, std::map<String, String>> CREATE_SCHEMA = {
-    {"required",{
-                    {"name", "string"}
-                }
+    std::map<String, std::map<String, String>> CREATE_SCHEMA = {
+        {"required",{
+                        {"name", "string"}
+                    }
+        }
+    };
+
+    void buildDatabaseArray(const Block & block, Poco::JSON::Object & resp)
+    {
+        Poco::JSON::Array databases_mapping_json;
+        if (block)
```

Comment:
we don't need this checking. if it is empty, `block.rows()` will be 0 ?

Reply:
fixed